package.json

```diff
@@ -135,103 +132,99 @@
     "tippy.js": "6.3.7",
-    "type-fest": "3.1.0",
+    "type-fest": "4.26.0",
     "typesafe-actions": "5.1.0",
-    "url-join": "4.0.1",
-    "uuid": "9.0.0",
-    "workbox-broadcast-update": "6.5.4",
-    "workbox-cacheable-response": "6.5.4",
-    "workbox-core": "6.5.4",
-    "workbox-expiration": "6.5.4",
-    "workbox-precaching": "6.5.4",
-    "workbox-routing": "6.5.4",
-    "workbox-strategies": "6.5.4",
-    "workbox-window": "6.5.4"
+    "url-join": "5.0.0",
+    "uuid": "10.0.0",
+    "workbox-broadcast-update": "7.1.0",
+    "workbox-cacheable-response": "7.1.0",
+    "workbox-core": "7.1.0",
+    "workbox-expiration": "7.1.0",
+    "workbox-precaching": "7.1.0",
+    "workbox-routing": "7.1.0",
+    "workbox-strategies": "7.1.0",
+    "workbox-window": "7.1.0"
   },
   "devDependencies": {
-    "@babel/core": "7.19.6",
-    "@babel/parser": "7.24.7",
-    "@babel/plugin-transform-runtime": "7.19.6",
-    "@babel/preset-env": "7.19.4",
-    "@babel/preset-react": "7.18.6",
-    "@babel/preset-typescript": "7.18.6",
+    "@babel/core": "7.25.2",
+    "@babel/parser": "7.25.6",
+    "@babel/preset-env": "7.25.4",
+    "@babel/preset-react": "7.24.7",
+    "@babel/preset-typescript": "7.24.7",
     "@svgr/webpack": "5.5.0",
-    "@testing-library/jest-dom": "6.4.2",
-    "@testing-library/react": "14.2.1",
-    "@testing-library/react-hooks": "8.0.1",
+    "@testing-library/dom": "10.4.0",
+    "@testing-library/jest-dom": "6.5.0",
+    "@testing-library/react": "16.0.1",
     "@testing-library/user-event": "14.5.2",
     "@types/d3": "5.16.3",
-    "@types/deep-freeze": "0.1.2",
-    "@types/gtag.js": "0.0.12",
-    "@types/jest": "29.2.1",
-    "@types/lodash-es": "4.17.6",
-    "@types/marked": "4.0.7",
-    "@types/react": "18.2.55",
-    "@types/react-dom": "18.2.0",
+    "@types/deep-freeze": "0.1.5",
+    "@types/gtag.js": "0.0.20",
+    "@types/jest": "29.5.12",
+    "@types/lodash-es": "4.17.12",
+    "@types/marked": "6.0.0",
+    "@types/react": "18.3.5",
+    "@types/react-dom": "18.3.0",
     "@types/react-router-dom": "5.3.3",
-    "@types/sanitize-html": "2.6.2",
-    "@types/url-join": "4.0.1",
-    "@types/uuid": "8.3.4",
-    "@typescript-eslint/eslint-plugin": "5.42.0",
-    "@typescript-eslint/parser": "5.42.0",
-    "assert": "2.0.0",
-    "axios-mock-adapter": "1.22.0",
-    "babel-jest": "29.2.2",
-    "babel-loader": "9.0.1",
+    "@types/sanitize-html": "2.13.0",
+    "@types/uuid": "10.0.0",
+    "@typescript-eslint/eslint-plugin": "^8.3.0",
+    "@typescript-eslint/parser": "^8.3.0",
+    "assert": "2.1.0",
+    "axios-mock-adapter": "2.0.0",
+    "babel-jest": "29.7.0",
+    "babel-loader": "9.1.3",
     "babel-plugin-dynamic-import-node": "2.3.3",
     "buffer": "6.0.3",
     "circular-dependency-plugin": "5.2.2",
-    "copy-webpack-plugin": "11.0.0",
-    "css-loader": "6.7.1",
-    "eslint": "8.26.0",
+    "copy-webpack-plugin": "12.0.2",
+    "css-loader": "7.1.2",
+    "eslint": "^8",
```

Comment:
The latest published version here is 9 something, but the switch to 9 is breaking and needs a big change in the config, so that's why I opted to say ^8, meaning the latest 8 version in case there are still small updates, without crossing the 9 boundary that would make it break.
I believe whenever we manage to get to the latest version we should go back to using the exact version.
Alternatively, I can just figure out what the latest 8 version is a explicitly write it as exact version, I doubt there would be many more 8 releases now that 9 is out.

Reply:
I agree 8/8.0.0 is not the right thing to do. When I do upgrades I search package.json for the `^` character and remove it. I think the idea is to keep the `node_modules` as consistent as possible on different people's machines. If you don't think it matters in this instance happy to leave with the `^`.